### ISSUE TITLE
Some issues regarding fine-tuning models

### ISSUE THREAD

**Issue by rainbownmm** [SalesforceAIResearch/uni2ts#11]
Hello, thank you for your excellent model and work. I have a question about fine-tuning the model. If I have about 1M of data that needs to be adjusted, what method should I use to build a dataset to fine tune the model? I tried your command line method, and by the way, I found that the loss could not be reduced. After adjusting the batch size, it decreased. If I want to adjust the learning rate and other parameters, where should I modify them? I see that the default configuration in the fine-tuning folder does not have this parameter

**Comment by liu-jc** [comment]
Hi @rainbownmm , for the learning rate and other parameters, you may find them in the model configuration, such as: https://github.com/SalesforceAIResearch/uni2ts/blob/667356adf0040adde8aefc95b70d99e8c737ac97/cli/conf/finetune/model/moirai_1.0_R_base.yaml#L25
Please let me if that answers your question.

**Comment by rainbownmm** [comment]
Thanks for the answer. I will try it. By the way, if there is 1M of data for fine-tuning, can the way of processing the data be processed and fine-tuned using the following command line?
python -m uni2ts.data.builder.simple ETTh1 dataset/ETT-small/ETTh1.csv --dataset_type wide

**Comment by gorold** [comment]
Hey, 1M of data is fine, but it mainly depends on your dataset format. `--dataset_type` supports [long or wide](https://stefvanbuuren.name/fimd/sec-longandwide.html) formats, with the first column being a parsable date/time format.

**Comment by rainbownmm** [comment]
Thanks for the answer.
